src/main/resources/i18n/condec.properties

```diff
@@ -79,7 +80,7 @@ condec.label.target.element=Target Element
 condec.label.unlinked.elements=Unlinked Elements
 condec.label.argument.type=Argument Type
 condec.label.link.type=Link Type
-condec.label.link.types=Link Types

```

Comment:
This might still be needed for the filter setting for link types.

Reply:
Thank you for letting me know! Reverted this change in the latest commit.